### PR TITLE
analytic content TrendMicro CAS

### PR DESCRIPTION
**Pull request by spsocprime**
## Proposed Changes

 Analytic content for TrendMicro CAS

### CODE REVIEW

**Comment by NikTripathi** [comment]
@aprakash13 Please review analytic rules. Thanks.

**Comment by v-sabiraj** [comment]
**@spsocprime** There should not be 2 templates with the same ID, id **c8e2ad52-bd5f-4f74-a2f7-6c3ab8ba687a** is duplicated. Please update it.

**Comment by v-sabiraj** [comment]
Workbooks Queries are failing. Please look into it.
image

**Comment by spsocprime** [comment]
> Workbooks Queries are failing. Please look into it. image

Error states that there are two functions with the same name in your environment. Probably renaming one will resolve this issue for you

**Comment by v-sabiraj** [comment]
@aprakash13  Please review Analytic Rules. Thanks.

**Comment by v-sabiraj** [comment]
Please fix the tactics "Exfitration" to "Exfiltration" in hunting queries of "Trend Micro CAS - DLP violations". Thanks.

**Comment by v-sabiraj** [comment]
image
Please fix the query, thanks.

**Comment by NikTripathi** [comment]
@aprakash13 Please review Analytic rules. Thanks

**Review by NikTripathi — COMMENTED** [review]
Workbook looks good. Can merge once APrakash approves.

**Review by NikTripathi — APPROVED** [review]
Workbook looks good. Can merge once APrakash approves.

File: Solutions/Trend Micro Cloud App Security/Hunting Queries/TrendMicroCASVAThreats.yaml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+id: 5ce1415f-cdea-4740-a481-73c1394248c2
+name: Trend Micro CAS - Virtual Analizer threats
```
Comment (aprakash13) [inline Solutions/Trend Micro Cloud App Security/Hunting Queries/TrendMicroCASVAThreats.yaml:2]: Is there a typo in Analizer. Should it be analyzer.

File: Solutions/Trend Micro Cloud App Security/Hunting Queries/TrendMicroCASVAThreats.yaml [hunk 2]
```diff
@@ -0,0 +1,25 @@
+id: 5ce1415f-cdea-4740-a481-73c1394248c2
+name: Trend Micro CAS - Virtual Analizer threats
+description: |
+  'Query searches for Virtual Analizer threats.'
```
Comment (aprakash13) [inline Solutions/Trend Micro Cloud App Security/Hunting Queries/TrendMicroCASVAThreats.yaml:4]: Is there a typo in Analizer. Should it be analyzer.

**Review by aprakash13 — COMMENTED** [review]
Overall, the queries are good.  A couple of typos probably needs to be fixed and then this should be ready to merge from query perspective.

File: Solutions/Trend Micro Cloud App Security/Hunting Queries/TrendMicroCASVAThreats.yaml (hunk 1 above)
Comment (spsocprime) [reply Solutions/Trend Micro Cloud App Security/Hunting Queries/TrendMicroCASVAThreats.yaml]: fixed

File: Solutions/Trend Micro Cloud App Security/Hunting Queries/TrendMicroCASVAThreats.yaml (hunk 2 above)
Comment (spsocprime) [reply Solutions/Trend Micro Cloud App Security/Hunting Queries/TrendMicroCASVAThreats.yaml]: fixed

**Pull request merged**